README.md

```diff
@@ -65,6 +65,31 @@ deployer also serves as a CI agent.
     VPC and routing table of the deployer.
 10. Run `scripts/deploy.sh <name>` to deploy the cluster.
 
+## Setting up continuous delivery
+
+Continuous Delivery is conceptually the same as continuous integration: a
+pipeline is triggered on BuildKite, which makes the associated BK agent run some
+commands. The same agent runs both CI and CD for disciplina, and it runs on the
+deployer.
+
+An agent associates itself with a BK project by providing it with a secret token.
+
+The deployment pipeline can be found here: https://buildkite.com/disciplina/disciplina-deploy
+
+When this particular pipeline is triggered, it will check out the associated Git
+repository, import `.buildkite/pipeline.deploy.yaml` and run the steps defined therein.
+
+This repository should contain all things necessary for a deployment, one way or
```

Comment:
I'm not sure CI should have write access to the repo, to be honest.

Reply:
In this case, you'd probably want to use the github api with a specific permission for writing tags.

Reply:
Maybe. But that seems like it's out of scope for the readme PR right now. Make an issue on youtrack describing what you have in mind so that we may discuss and plan in a more organized fashion.